RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -64,6 +78,11 @@ public struct CustomerCenterView: View {
             await loadInformationIfNeeded()
         }
         .environmentObject(self.viewModel)
+        .onAppear {
```

Comment:
Nit: It might be simpler to do this initialization where `_navigationPathWrapper` is declared instead of here in an onAppear. Something like:

```swift
@State private var _navigationPathWrapper: Any? = {
    if _navigationPathWrapper == nil, #available(iOS 16.0, *) {
                return NavigationPathWrapper()
    } else {
        return nil
   }
}()
```